includes/class-wc-stripe-intent-controller.php

```diff
@@ -875,10 +875,14 @@ private function validate_payment_intent_required_params( $required_params, $non
 
 		// Bail out if we're missing required information.
 		if ( ! empty( $missing_params ) ) {
+			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_debug_backtrace
+			$calling_method = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS, 2 )[1]['function'] ?? '';
```

Comment:
Quick question, why do we need to log the calling method? 🤔

Reply:
Not exactly needed, but I thought it could be useful when debugging since this method can be called by either `create_and_confirm_payment_intent` or `update_and_confirm_payment_intent`.